### PR TITLE
Press hits edit

### PR DESCRIPTION
**Pull request by amwhitty**
adding new press hits to homepage and news page

### CODE REVIEW

**Comment by TomNUSDS** [comment]
Here's what it looks like:

image

File: images/press-logo-nextgov.svg
Comment (TomNUSDS) [inline images/press-logo-nextgov.svg:1]: These svgs seem a bit oversized? I have a trick to fix them I can show you.

Comment (amwhitty) [reply images/press-logo-nextgov.svg:1]: Yes please let me know @TomNUSDS. I just sized the new ones to match the existing. Open to improvements.

**Review by TomNUSDS — APPROVED** [review]
LGTM

**Pull request merged**